src/types.d.ts

```diff
@@ -42,7 +52,7 @@ export interface JestPlaywrightConfig {
   contextOptions?: BrowserContextOptions
   exitOnPageError: boolean
   browsers: BrowserType[]
-  devices?: string[]
+  devices?: string[] | CustomDeviceType[]
```

Comment:
defining devices as `string[] | CustomDeviceType[]` results in devices needing to be all strings or all `CustomDeviceType`

For an array containing either strings or CustomDeviceType you should use

`devices?: (string | CustomDeviceType)[]`